redis/src/commands/mod.rs

```diff
@@ -191,12 +191,12 @@ implement_commands! {
     }
 
     /// Rename a key.
-    fn rename<K: ToRedisArgs>(key: K, new_key: K) {
+    fn rename<K: ToRedisArgs, N: ToRedisArgs>(key: K, new_key: N) {
```

Comment:
Is there any specific reason to change this? 

If not, I think we should limit the changes to where we actually need them, which from what I can tell would be a few commands like `SUNIONSTORE` that doesn't specify the 2nd argument as either a different generic or a &[K].